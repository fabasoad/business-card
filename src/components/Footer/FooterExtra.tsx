import * as React from 'react'
import Iframe from 'react-iframe'

export default function FooterExtra() {
  return (
    <div className="d-flex justify-content-center">
      <Iframe
        url={'https://www.worldcommunitygrid.org/getDynamicImage.do?memberName=fabasoad&mnOn=true&stat=3&imageNum=1&rankOn=false&projectsOn=false&special=false&link=1&memberId=1103124'}
        frameBorder={0}
        name="di"
        scrolling="no"
        width="405px"
        height="145px"
      />
    </div>
  )
}
