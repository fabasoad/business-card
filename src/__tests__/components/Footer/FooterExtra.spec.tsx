/// <reference types="jest" />
import * as React from 'react'
import { shallow } from 'enzyme'
import FooterExtra from '../../../components/Footer/FooterExtra'

test('should render FooterExtra correctly', () =>
  expect(shallow(<FooterExtra />)).toMatchSnapshot()
)
