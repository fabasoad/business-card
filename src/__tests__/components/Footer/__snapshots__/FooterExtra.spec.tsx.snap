// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FooterExtra correctly 1`] = `
<div
  className="d-flex justify-content-center"
>
  <Iframe
    frameBorder={0}
    height="145px"
    name="di"
    scrolling="no"
    url="https://www.worldcommunitygrid.org/getDynamicImage.do?memberName=fabasoad&mnOn=true&stat=3&imageNum=1&rankOn=false&projectsOn=false&special=false&link=1&memberId=1103124"
    width="405px"
  />
</div>
`;
